Add selected-cipher-suite default behavior (#204)
ietf-teep/teep-protocol: draft-ietf-teep-protocol.md

  QueryRequest.

selected-cipher-suite
: The selected-cipher-suite parameter indicates the selected ciphersuite. If this
  parameter is not present, it is to be treated as if the TEEP Agent accepts

>   any ciphersuites listed in the QueryRequest, and the TAM can select it.

Reviewer: ```suggestion
  any ciphersuites listed in the QueryRequest, so the TAM can select one.
```
Another wordsmith suggestion

>   any ciphersuites listed in the QueryRequest, and the TAM SHOULD select it.

Reviewer: ```suggestion
  any ciphersuites listed in the QueryRequest, and the TAM can select it.
```
There's no obvious alternative to the SHOULD, so change language.

(My preference would actually me to make the selected-cipher-suite mandatory instead of optional though, but I am ok either way.)
